### DISCUSSION TITLE
[Feature] Option to show jobs stacked in multiple columns

### DISCUSSION THREAD

**Discussion by bmondream** [immich-app/immich#5402] [Feature Request]
### Option to show jobs stacked in multiple columns

First of all, this is a wonderful project and I can't appreciate it enough!

When checking running jobs in the Jobs Status page, I think it would be nicer and clearer to display more job boxes all at once.

The boxes themselves can, I believe, be a little narrower without breaking UX. Split them into (a fixed amount? of) columns so more of them can fit on a single screen and you eliminate the need for scrolling. I imagine there are some people that would appreciate this enhancement.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by Zonmonbon** [comment]
Came to propose this idea, glad to see it here already!

**Comment by kneutron** [comment]
I was able to get this working at commandline using the REST API, see comment on #22241
